src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BytesKmerizer.java

```diff
@@ -0,0 +1,78 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import java.util.Iterator;
+import java.util.NoSuchElementException;
+
+/**
+ * Iterator over successive Kmers from a byte[].
+ * Bytes are interpreted as ASCII characters.
+ * Silently skips over parts of the sequence that has characters other than A, C, G, or T.
+ *
+ * Created by tsharpe on 12/18/15.
+ */
+public class BytesKmerizer implements Iterator<Kmer>
```

Comment:
Is there a reason this shouldn't be `Iterable`? It seems like it could easily be made repeatable.  If it isn't made iterable I would rename it to KmerIterator  or something along those lines.